New package: CatPalette v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: CatPalette
- Repository: https://github.com/lyriath-love/CatPalette
- Created by: @lyriath-love
- Version: v0.1.0
- Commit: 7cd919fe3fe48c63dba070b1007f596ddfdd514a
- Git reference: HEAD

tecosaur commented:
@lyriath-love just out of curiosity, have you seen Crayons.jl or StyledStrings.jl?

JuliaHubRegistrator committed:
New package: CatPalette v0.1.0

UUID: 071649ef-635e-4d97-928b-23bc9a7e4a6f
Repo: https://github.com/lyriath-love/CatPalette.git
Tree: 926d7461fa188e786f7ed7efa7014be2c3939c6d

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

tecosaur commented:
You also seem to have included the project Manifest, which isn't right.

[noblock]
